README.md

```diff
@@ -1 +1,60 @@
-# capstone-project-1
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Web Development Course Capstone Project
+
+> This is my capstone project website showcasing my skills and mastery of HTML, CSS, JAVASCRIPT and responsive design.
+
+![Capstone project](img/codeniversity_desktop2.PNG)
+
+
+## Built With
+
+- HTML
+- CSS
+- JAVASCRIPT
+
+## Live Demo 🔗
+
+[Live Demo Link](https://code-panther.github.io/capstone-project-1/)
+
+
+## Getting Started
+
+**This is my capstone project website showcasing my skills and mastery of HTML, CSS, JAVASCRIPT and responsive design, showing proficiency in the programming languages listed above**
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+There are no prerequisites
+
+### Setup
+Just clone the repository to your local machine and you are all set.
+`git clone https://github.com/Code-panther/Portfolio.git`
+
+
+
+## Authors
+
+👤 **Author**
```

Comment:
- [ ] OPTIONAL
Kindly replace the text **Author** with your full name, so as to make your ReadMe more professional

Reply:
Thank you for this observation. I will do just that.